garageScript/c0d3-app
# feat(admin): Create challenges page

Related to #2081 

### Description

This PR aims at adding the page that'll be used to create and edit the challenges.

<details>
<summary>
Page image
</summary>
<img width="1380" alt="image" src="https://user-images.githubusercontent.com/35906419/200117103-f956d51d-9d5e-4ec6-badf-57ac3943a36e.png">
</details>


### Testing

1. Go to `/admin/lessons/js0/challenges`
2. Select one of the challenges in the sidebar and click it
3. Edit the challenge name
4. Press `SAVE CHANGES` → Make sure it got updated correctly
5. Press `ADD NEW MODULE` at the bottom of the sidebar
6. Fill in the inputs and set the `order` as 0 → Make sure it appears as the first item in the sidebar

### Issues

The sidebar text should be `ADD NEW CHALLENGE` & `NO CHALLENGES` instead of modules. This will be fixed in a subsequent PR.
